### PR TITLE
Update sidekiq.rb

### PR DESCRIPTION
**Pull request by seuros**
instead of testing Sidekiq version, let use gem to select the correct version

### CODE REVIEW

**Comment by sheerun** [comment]
Is it RubyGems feature?

**Comment by seuros** [comment]
no, Ruby.
rubygems/core_ext/kernel_gem.rb

File: lib/airbrake/sidekiq.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
+gem 'sidekiq', '~> 3.0'
 require 'sidekiq'
 
-if ::Sidekiq::VERSION >= '3'
-  ::Sidekiq.configure_server do |config|
-    config.error_handlers << lambda do |exception, context|
-      Airbrake.notify_or_ignore(exception, :parameters => context)
-    end
+Sidekiq.configure_server do |config|
+  config.error_handlers << ->(exception, context)
```
Comment (shifi) [inline lib/airbrake/sidekiq.rb]: @seuros did you missed `do` at the end of the line?
`  config.error_handlers << ->(exception, context) do`
Without it I get syntax error.

Comment (shifi) [inline lib/airbrake/sidekiq.rb]: Also, I think lambda literal syntax is better when you have single line block, but I guess that is question of personal taste :)

Comment (seuros) [inline lib/airbrake/sidekiq.rb]: I will revert to lambda syntax then. Thank you

Comment (shifi) [inline lib/airbrake/sidekiq.rb]: Thanks

Comment (shifi) [inline lib/airbrake/sidekiq.rb]: @seuros I changed that for you. Thank you!

**Pull request merged**